tests/tokenizer.test.js

```diff
@@ -0,0 +1,72 @@
+const tokenize = require('../src/tokenizer.js').tokenize;
+
+const tests = [
+[
+`echo "Hello, World!"`,
+'[{\"col\":0,\"line\":1,\"type\":\"Identifier\",\"value\":\"echo\"},{\"col\":4,\"line\":1,\"type\":\"Operator\",\"value\":\"\\\"Hello, World!\\\"\"}]'
```

Comment:
Can we write these as objects and stringify later if required? This is illegible.

Reply:
I am positive that we can find a way to write these objects in a better way that it much more legible, however, this will need to wait until cleaning time.

Reply:
Let's take the time to do this right the first time instead of backing ourselves into a hole we need to clean ourselves out of down the line. I can't approve test cases that I can't really verify are correct to begin with.